src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,33 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
+    @SuppressWarnings({"checkstyle:Indentation", "checkstyle:WhitespaceAround"})
```

Comment:
Nie ignoruj checkstyle. Usuń tę adnotację i jeżeli są jakieś błędy, np. zbędne entery, albo złe wcięcia, to sformatuj odpowiednio kod.